paws/templates/legacy.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    name: paws-public-legacy
+    toolforge: tool
+  annotations:
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      rewrite /paws-public(/|$)*(.*) $scheme://public.paws.wmcloud.org/$2 redirect;
```

Comment:
Is there an advantage here over using the existing capture group defined by the spec.rules section below? If not this can be simplified by using a "permanent-redirect" annotation which inherits the capture groups from spec.rules:
```yaml
nginx.ingress.kubernetes.io/permanent-redirect: https://public.paws.wmcloud.org/$2
```

Reply:
The advantage is just that I couldn't make that work. I was trying for a temporal redirect in case there's a mistake there as well, so I tested with using nginx.ingress.kubernetes.io/temporal-redirect

I can go back and try again because I also tried other things in the process :)